pkg/graveler/graveler_test.go

```diff
@@ -659,6 +663,16 @@ func TestGraveler_PreMergeHook(t *testing.T) {
 			if err != nil && !errors.As(err, &hookErr) {
 				t.Fatalf("Merge err=%v, pre-merge error expected HookAbortError", err)
 			}
+			if refManager.AddedCommit.MetaRangeID == "" {
+				t.Fatalf("Empty MetaRangeID, commit was successful")
```

Comment:
Can you log the entire commit?  It may have clues.  (And if not, just `t.Fatal`).